src/util/StringUtils.h

```diff
@@ -381,6 +385,33 @@ vector<string> split(const string& orig, const char sep) {
   return result;
 }
 
+// _____________________________________________________________________________
+vector<string> splitWs(const string &orig) {
+  vector<string> result;
+  if (orig.size() > 0) {
+    size_t start = 0;
+    size_t pos = 0;
+    while (pos < orig.size()) {
+      if (orig[pos] >= 0 && ::isspace(orig[pos])) {
```

Comment:
I got a little obsessed with this and tried it out on my Raspberry Pi where `char` is unsigned. Your code definitely works and even just `::isspace(orig[pos])` works with both `-fsigned-char` and `-funsigned-char`.
So I feel like doing an extra condition is a bit too much just to make it clear that we expect the chars to use the full 8-bits. I think `::isspace(static_cast<unsigned char>(orig[pos]))` might be a good compromise. It shows we were thinking about 8-bit chars and it matches the `::isspace()` documentation [representable as an unsigned char](http://en.cppreference.com/w/c/string/byte/isspace).